frontend/src/pages/transactions/[hash].jsx

```diff
@@ -55,13 +57,16 @@ class TransactionDetailsPage extends React.Component {
             />
           </Content>
         )}
-        {this.props.receiptsOutcome && (
+
+        {receiptsOutcome && (
```

Comment:
You should check for `this.props.receipts`, otherwise, it looks odd since you don't use `receiptsOutcome`